guide/popular-topics/reactions.md

```diff
@@ -155,6 +155,127 @@ if (message.content === '!fruits') {
 
 The benefit of this small optimization is that you can use `.then()` to handle when all of the Promises have resolved, or `.catch()` when one of them has failed. You can also `await` it since it returns a Promise itself.
 
+## Removing reactions
+
+Now that you know how to add reactions, you might be asking, how do we remove them? In this section you will learn how to remove all reactions, remove reactions by user, and remove reactions by emoji.
+
+::: warning
+All of these methods require `MANAGE_MESSAGES` permissions. Make sure your bot has permissions before attempting to utilize any of these methods, as it will error if it doesnt.
+:::
+
+### Removing all reactions
+
+Removing all reactions from a message is the easiest, the API allows us to do this through a single call. It can be done through the <branch version="11.x" inline> `message.clearReactions()` </branch> <branch version="12.x" inline> `message.reactions.removeAll()` </branch> method. 
+
+<branch version="11.x">
+
+```js
+message.clearReactions()
+	.catch(err => console.error('Failed to clear reactions: ', err));
```

Comment:
This can be a one-liner, and `err` should be `error`.

Reply:
Woops, interpreted it as on top of the code block and inside of it

Reply:
That was for the eslint skip comment, god im a mess today